lib/widgets/video_viewer_dialog.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:fluffychat/pages/chat/events/message_content_style.dart';
+import 'package:fluffychat/utils/platform_infos.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:media_kit/media_kit.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:media_kit_video/media_kit_video.dart';
+
+class VideoViewerDialog extends StatefulWidget {
```

Comment:
```suggestion
class FullScreenVideoPlayer extends StatefulWidget {
```
then please create static function in this,
just call `FullScreenVideoPlayer.play(videoPath);` it will show full screen video player. done